src/pages/Main/Main.js

```diff
@@ -1,4 +1,3 @@
-import React from "react";
```

Comment:
> @sandrina-p I also created a useFormSubmit to store data from checkout, contact and newsletter forms. The 'submitted' state is used to render a div on form submit, but then it renders for all three forms. The question is, is there a way to use a single state or would I need a separate fooIsSubmited for each form? Does creating context around this feature make sense?

@majac91 , I did not understand your question exactly :/ I looked at the code, why do you have a `submitted` at the App level? You could create one  for each component.

Reply:
I lifted it thinking I could pass it down where needed and then realized a single state is not going to work but thought maybe there's a way around it. Separate states it is then.